code/modules/client/preferences.dm

```diff
@@ -57,6 +57,7 @@ GLOBAL_LIST_EMPTY(preferences_datums)
 	var/undershirt = "Nude"				//undershirt type
 	var/socks = "Nude"					//socks type
 	var/backbag = DBACKPACK				//backpack type
+	var/jumps = SUIT					//suit/skirt

```

Comment:
A more descriptive variable name, e.g. `jumpsuit_style` or even just `jumpsuit` would be better than `jumps`.